Client/metrics.yaml

```diff
@@ -630,9 +630,8 @@ app:
   opened_as_default_browser:
     type: counter
     description: |
-      Recorded when a preference is changed and includes the
-      preference that changed as well as the value changed to
-      recorded in the extra keys.
+      Records the number of times that the application was opened
+      as the default browser.
```

Comment:
@magorlick does this match your observations of the metric behaviour? I checked the code and it seems right to me.

https://github.com/mozilla-mobile/firefox-ios/blob/main/Client/Application/NavigationRouter.swift#L110

Reply:
This is correct. I was going to write "Counts the number of times the app is opened from an external link. This implies
      the user has Firefox set as a default browser. Currently this is our most accurate way
      of measuring for default browser." to add some more context, if that is helpful!

Reply:
Awesome, thanks @dnarcese! Your description sounds better, I'll adjust it.

Reply:
@dnarcese I adjusted the description to mostly be what you suggested (with some minor stylistic tweaks). Let me know if I need to do anything else.

Reply:
@wlach glean linter is complaining about trailing spaces at (635:1)

Reply:
Should hopefully be fixed now, sorry!